src/traits.rs

```diff
@@ -351,6 +351,10 @@ pub trait HttpContext: Context {
         hostcalls::add_map_value_bytes(MapType::HttpRequestHeaders, name, value).unwrap()
     }
 
+    fn remove_http_request_header(&self, name: &str) {
```

Comment:
Could you add those for request trailer, response header and response trailer as well? Thanks!

Reply:
Friendly ping.